src/MusicStore/MusicStore.csproj

```diff
@@ -59,9 +59,6 @@
     <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel">
       <Version>1.2.0-*</Version>
     </PackageReference>
-    <PackageReference Include="Microsoft.AspNetCore.Server.WebListener">
-      <Version>0.1.0</Version>
-    </PackageReference>
```

Comment:
This is a super old version of this package and none of this code will be executed on the path that we're testing here. It seemed better to just remove this and with it a bunch of compilation from `Main()`